### ISSUE TITLE
Drupal best practice ruleset can't be executed

### ISSUE THREAD

**Issue by adrian-cid** [pfrenssen/coder#18]
I'm having this problem with this command:

```
phpcs -ps --colors --standard=~/.composer/vendor/drupal/coder/coder_sniffer/DrupalPractice/ruleset.xml --extensions='php,module,inc,install,test,profile,theme,css,info,txt,md'
```
After this update:

```
  - Updating squizlabs/php_codesniffer (2.9.1 => 3.3.2): Loading from cache
  - Updating drupal/coder (8.2.12 => 8.3.1):  Checking out 29a25627e7
```

**Comment by pfrenssen** [comment]
It looks like we need to apply our custom autoloader also to the DrupalPractice ruleset. I have proposed a fix in #19, could you check if this fixes the issue for you?

**Comment by adrian-cid** [comment]
#19 fix the problem.

**Comment by pfrenssen** [comment]
Thanks for testing! The PR fails the test suite though so this needs some more work.

**Comment by klausi** [comment]
Fixed in #29
